assembly/guide-configs/app-dev/features.json

```diff
@@ -1,10 +1,10 @@
 {
   "file_name": "features.md",
-  "file_dir": "application-development",
+  "file_dir": "dev-guide",
```

Comment:
I think for the slug, I want to see the full `developer-guide`

For the sidebar labels I am good with "Dev guide"

For the page titles, I want "Temporal Application developer's guide - foundations" etc...